tests/subsys/ipc/pbuf/prj.conf

```diff
@@ -5,3 +5,4 @@ CONFIG_ZTEST_SHUFFLE=y
 CONFIG_IPC_SERVICE=y
 CONFIG_IPC_SERVICE_ICMSG=y
 CONFIG_PBUF=y
+CONFIG_PBUF_CACHE_HANDLING=n
```

Comment:
Why bother with the new option and changes in the pbuf implementation instead of 
```suggestion
# pbuf is designed to run writer and reader code on two CPUs.
# The write function cannot be preempted by the read function in a CPU,
# or the cache management could result in data corruption.
# In the tests, the reader and the writer are executed on a single CPU.
# Disable data cache to prevent potential data corruption.
CONFIG_DCACHE=n
```

Reply:
Thanks. With the comment, it's now obvious to the reader why the cache is disabled and why data corruption is expected with the cache enabled.